app/uk/gov/hmrc/plasticpackagingtax/registration/views/landing_page.scala.html

```diff
@@ -0,0 +1,57 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import uk.gov.hmrc.plasticpackagingtax.registration.controllers.routes._
+@import uk.gov.hmrc.plasticpackagingtax.registration.views.html.components._
+@import uk.gov.hmrc.plasticpackagingtax.registration.views.html.main_template
+@import uk.gov.hmrc.plasticpackagingtax.registration.views.model.Title
+@import uk.gov.hmrc.plasticpackagingtax.registration.views.model.TaskListViewModel
+
+
+@this(
+govukLayout: main_template,
+pageTitle: pageTitle,
+link: link,
+paragraphBody: paragraphBody,
+govukTag: GovukTag,
+govukButton: GovukButton,
+taskList: taskList
+)
+
+@(tasks: Seq[TaskListViewModel])(implicit request: Request[_], messages: Messages)
+
+@govukLayout(title = Title("landingPage.title")) {
+
+<div class="govuk-!-width-two-thirds">
+    @pageTitle(text = messages("landingPage.title"))
+</div>
+
+<div id="contents">
```

Comment:
Similar do this `div`. 
We should use GDS Design System as much as possible.

Reply:
removed